### ISSUE TITLE
Server Code

### ISSUE THREAD

**Issue by Yu-Vitaqua-fer-Chronos** [nothjarnan/chatter-cc#4]
Heya @nothjarnan, is the code still available for the server?

**Comment by nothjarnan** [comment]
Yes, I'm not home at the moment but I'll see if I can find it tomorrow.

**Comment by The-Mythical-Forest-Collective** [comment]
Thanks! Is it also cool to ask if you host the code on GitHub in this repo so it can be grabbed easily? Stumbled upon the post on CC forums and was interested in it, I'm sure it's the same for many others too

**Comment by nothjarnan** [comment]
Hey,
Sorry for the slow reply. Got caught up in midsummer celebrations.
I've been scouring through my computer and unfortunately I can't find the original server code. Chances are it has been lost though the many reformats over the years. 
If you're feeling brave, you are much welcome to write a new server based on what the client sends to the server.

**Comment by The-Mythical-Forest-Collective** [comment]
No worries! And thanks anyway aha
